3-networks-dual-svpc/modules/base_env/main.tf

```diff
@@ -36,111 +36,141 @@ locals {
-    "adsdatahub.googleapis.com",
     "aiplatform.googleapis.com",
     "alloydb.googleapis.com",
-    "alpha-documentai.googleapis.com",
     "analyticshub.googleapis.com",
     "apigee.googleapis.com",
     "apigeeconnect.googleapis.com",
+    "apikeys.googleapis.com",
+    "apphub.googleapis.com",
     "artifactregistry.googleapis.com",
-    "assuredworkloads.googleapis.com",
     "automl.googleapis.com",
+    "backupdr.googleapis.com",
     "baremetalsolution.googleapis.com",
     "batch.googleapis.com",
+    "beyondcorp.googleapis.com",
+    "biglake.googleapis.com",
     "bigquery.googleapis.com",
     "bigquerydatapolicy.googleapis.com",
     "bigquerydatatransfer.googleapis.com",
     "bigquerymigration.googleapis.com",
-    "bigqueryreservation.googleapis.com",
     "bigtable.googleapis.com",
     "binaryauthorization.googleapis.com",
+    "blockchainnodeengine.googleapis.com",
+    "certificatemanager.googleapis.com",
     "cloud.googleapis.com",
+    "cloudaicompanion.googleapis.com",
     "cloudasset.googleapis.com",
     "cloudbuild.googleapis.com",
-    "clouddebugger.googleapis.com",
+    "cloudcontrolspartner.googleapis.com",
     "clouddeploy.googleapis.com",
     "clouderrorreporting.googleapis.com",
     "cloudfunctions.googleapis.com",
     "cloudkms.googleapis.com",
     "cloudprofiler.googleapis.com",
-    "cloudresourcemanager.googleapis.com",
+    "cloudquotas.googleapis.com",
     "cloudscheduler.googleapis.com",
     "cloudsearch.googleapis.com",
+    "cloudtasks.googleapis.com",
     "cloudtrace.googleapis.com",
     "composer.googleapis.com",
     "compute.googleapis.com",
+    "confidentialcomputing.googleapis.com",
+    "config.googleapis.com",
     "connectgateway.googleapis.com",
+    "connectors.googleapis.com",
     "contactcenterinsights.googleapis.com",
     "container.googleapis.com",
     "containeranalysis.googleapis.com",
     "containerfilesystem.googleapis.com",
     "containerregistry.googleapis.com",
+    "containersecurity.googleapis.com",
     "containerthreatdetection.googleapis.com",
+    "contentwarehouse.googleapis.com",
     "datacatalog.googleapis.com",
     "dataflow.googleapis.com",
+    "dataform.googleapis.com",
     "datafusion.googleapis.com",
+    "datalineage.googleapis.com",
     "datamigration.googleapis.com",
+    "datapipelines.googleapis.com",
     "dataplex.googleapis.com",
     "dataproc.googleapis.com",
     "datastream.googleapis.com",
-    "dialogflow.googleapis.com",
+    "discoveryengine.googleapis.com",
     "dlp.googleapis.com",
     "dns.googleapis.com",
     "documentai.googleapis.com",
-    "domains.googleapis.com",
+    "earthengine.googleapis.com",
+    "essentialcontacts.googleapis.com",
     "eventarc.googleapis.com",
     "file.googleapis.com",
+    "financialservices.googleapis.com",
     "firebaseappcheck.googleapis.com",
     "firebaserules.googleapis.com",
     "firestore.googleapis.com",
     "gameservices.googleapis.com",
-    "gkebackup.googleapis.com",
```

Comment:
`gkebackup.googleapis.com` is supported at GA level
https://cloud.google.com/vpc-service-controls/docs/supported-products

```
Backup for GKE
Status	Preview. The integration of this product with VPC Service Controls is in Preview and is ready for broader testing and use, but is not fully supported for production environments.
Protect with perimeters?	Yes. You can configure your perimeters to protect this service.
Service name	gkebackup.googleapis.com
Details	
You can use VPC Service Controls to protect backup for GKE and you can use backup for GKE features normally inside service perimeters.

For more information about Backup for GKE, refer to the product documentation.

Limitations	
The Backup for GKE integration with VPC Service Controls has no known limitations.
```